docs/root/configuration/access_log.rst

```diff
@@ -228,6 +228,13 @@ The following command operators are supported:
   Local address of the upstream connection. If the address is an IP address it includes both
   address and port.
 
+%UPSTREAM_TRANSPORT_FAILURE_REASON%
+  HTTP
+    The reason of upstream transport failure, this is usually upstream TLS connection failure with 503 response code.
```

Comment:
Can you make this a little more clear? Is the latter part of the sentence the actual error message? Should it be in quotes? Also, do we want a string here, or should we have well defined error codes like we do for other failures?

Reply:
I considered well defined error codes but I chose a string here due to the transport socket extensibility and it is likely only used in logging purpose. We can make some well known errors as predefined strings though. No the latter part is not the actual message, will update the document.